Source/VissimSimulator/CollectorWorker.cs

```diff
@@ -76,6 +76,7 @@ static void AddEvent(int locationId, int cellularTowerId, string eventType, stri
                 {
                     Console.WriteLine("Count not insert.");
                 }
```

Comment:
where is the catch clause here? try should be paired with catch and finally clause.